apps/docs-snippets2/sway/proxy/Forc.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "proxy"
+
+[dependencies]
+standards = { git = "https://github.com/FuelLabs/sway-standards", tag = "v0.6.0" }
+sway_libs = { git = "https://github.com/FuelLabs/sway-libs", tag = "v0.24.0" }
```

Comment:
We must be sure this is the correct version of this contract.

Can we please have a README explaining all these details?

This must always be paired with the one used by the `fuels` CLI, and `forc`: 
- https://github.com/FuelLabs/fuels-ts/tree/master/packages/fuels/src/cli/commands/deploy/proxy

Reply:
We could export it from `typegen` as like example templates?

Reply:
Exporting this has been a bit painful. 

If the generated factory exists inside `fuels` and we want to export it from there, we get cyclic deps for all the things that the factory itself is importing. So we'd need to swap out the imports for the module imports. 

It probably needs to exist in it's own package, that imports all the required module imports that the factory needs. With a script that swaps the `fuels` imports for the module imports so we can still export it from `fuels`.

Will feedback soon.

Reply:
@danielbate Yes. I've struggled to find a solution for this as well.

Reply:
I've made many new changes to this PR now, including the addition of `@fuel-ts/recipes` which should solve this problem, let me know